Pod/Classes/Format/CardNumberFormatter.swift

```diff
@@ -47,10 +47,10 @@ public final class CardNumberFormatter {
      
      - returns: Formatted card number string.
      */
-    public func formattedCardNumber(cardNumberString: String) -> String {
+    public func format(cardNumber: String) -> String {
```

Comment:
This matches what I mentioned above (`unformat(cardNumberString: String)`. Whatever function signature we decide to use, these two should be consistent.